src/components/textarea/textarea.stories.mdx

```diff
@@ -214,3 +218,34 @@ It is possible to use the `tooltipPosition` to override the default placement of
 <StyledSystemProps of={OriginalTextarea} noHeader margin />
 
 **Any other supplied props will be provided to the underlying HTML input element**
+
+<TranslationKeysTable
+  translationData={[
+    {
+      name: "characterCount.tooManyCharacters",
+      description:
+        "The message displayed below the input which will inform"
+        +" users if they have excedded the set `characterLimit` and how much by." 
+        +" This will also be announced to screenreaders.",
+      type: "func",
+      returnType: "string",
+    },
+        {
+      name: "characterCount.charactersLeft",
+      description:
+        "The message displayed below the input which will inform"
+        +" users how many characters they have before they reach  or exceed the"
+        +" set `characterLimit`. This will also be announced to screenreaders.",      
```

Comment:
**suggestion**

```suggestion
        +" users if they have exceeded the set `characterLimit` and how much by." 
        +" This will also be announced to screen readers.",
      type: "func",
      returnType: "string",
    },
        {
      name: "characterCount.charactersLeft",
      description:
        "The message displayed below the input which will inform"
        +" users how many characters they have before they reach  or exceed the"
        +" set `characterLimit`. This will also be announced to screen readers.",      
```